scheduler/domain/README.md

```diff
@@ -8,5 +8,5 @@ The scheduler code contains the following packages:
 * __domain__ - scheduler go objects: jobs, tasks and states, and thrift versions of these objects
   * __gen-go/sched__ - generated code from the thrift definitions
   * __scheduler__ - interfaces and implementations for job scheduling
-  * __worker__ - interface for scheduler to run tasks on a worker
+  * __worker__ - interface for scheduler to starts tasks on a worker & manage responses re: status of in progress tasks.
```

Comment:
s/starts/start/

Reply:
Also re: domain - worker sub-pkg doesn't feel like it belongs here either. Maybe a top-level pkg under scheduler, or within setup (looks related to setup/workers.go). + update this readme with changes